gen/enum.go

```diff
@@ -162,6 +163,20 @@ func enum(g Generator, spec *compile.EnumSpec) error {
 			return &<$v>
 		}
 
+		<$sw := newVar "sw">
+		// Encode encodes <$enumName> directly to the wire.
```

Comment:
Is to bytes more appropriate since 'the wire' could be confused for the intermediate representation to avoid?